unhide-game/index.md

```diff
@@ -0,0 +1,21 @@
+## How do I unhide/hide my game?
+
+This is it, the big moment. Once your game has a thumbnail and a maturity rating, you will be able to unleash it upon an unsuspecting public.
+
+But WAIT! Before you unhide your game, you can add media, music, news, and actual game builds to make your game page all that it can be.
+
+When you're ready, visit your [dashboard](http://gamejolt.com/dashboard/) and select the game from the list under "Your Games". You will be taken to the game's management page. Here, you can publish the game by clicking/tapping "Unhide your game."
```

Comment:
we need to update this now that you can Hide/Publish your game page vs Hide/Unhide. Also once you select the game from "Your games" it takes you to the overview page and from there you'll need to select "Edit Details" tab on the left nav and at the bottom under the Visibility settings is where you're able to hide/publish your game page